sycl/source/detail/windows_pi.cpp

```diff
@@ -22,7 +22,7 @@ void *loadOsLibrary(const std::string &PluginPath) {
 }
 
 void *getOsLibraryFuncAddress(void *Library, const std::string &FunctionName) {
-  return GetProcAddress((HMODULE)Library, FunctionName.c_str());
+  return (void*)GetProcAddress((HMODULE)Library, FunctionName.c_str());
```

Comment:
This is to correct this:
xpti_proxy.cpp(116,14): error: implicit conversion between pointer-to-function and pointer-to-object is a Microsoft extension [-Werror,-Wmicrosoft-cast]
      return m_dispatch_table[index];
             ^~~~~~~~~~~~~~~~~~~~~~~